src/Client.php

```diff
@@ -303,9 +303,9 @@ public function getAccountInfo(): array
      *
      * @return array
      */
-    public function revokeToken(): array
+    public function revokeToken()
     {
-        return $this->rpcEndpointRequest('auth/token/revoke', []);
+        return $this->voidEndpointRequest('auth/token/revoke');
```

Comment:
@freekmurze Done, please checkout https://github.com/spatie/dropbox-api/pull/19/commits/febeb488d1f0756018c51d863d8c6227b7127f50

It was tested both on local and production environment, and CI is green.